classes/messenger/factories/course_recipient_send/message_recipient_send_factory.php

```diff
@@ -128,12 +128,17 @@ private function send_message_to_mentor_user($mentor_user, $mentee_user)
     public function send_to_mentors()
     {
         $mentor_users = $this->get_recipient_mentors();
+        $prepended_mentor_users = $this->get_recipient_prepended_mentors();
 
         $mentee_user = $this->recipient->get_user();
 
         foreach ($mentor_users as $mentor_user) {
             $this->send_message_to_mentor_user($mentor_user, $mentee_user);
         }
+
+        foreach($prepended_mentor_users as $mentor_user) {
```

Comment:
`send_message_to_mentor_user` won't work for fake accounts. Would be better to use `send_email_to_mentor_user` for profile field mentors when sending as a moodle message.